instructions.md

```diff
@@ -0,0 +1,203 @@
+- A successful submission should clear the form.
+
+### Hint
+
+- Add `{ useState, useEffect }` to the `'react'` import statement
+- Import `ContactForm` and `TileList`
+- Make `props` a function parameter
+- The contact info state variables should default to an empty string using `useState('')`
+- The state variable to track a duplicate name should be set to false using `useState(false)`
+- Implement a call to `useEffect`, where the callback function sets the duplicate state variable to `true` if the name state variable is already in the contacts list
+- An `if` statement should be added to `handleSubmit` that checks the duplicate state variable is `false`. The body of the `if` statement should call the add contact function in `props` and set the contact info state variables to `''` 
+
+### Task #4 
+
+Based on the given requirements, implement `ContactForm` as a stateless component that renders a web form to collect the necessary contact information.
+
+This component will:
+- Have 7 props: 
+  - The variables for contact name, phone and email 
+  - The setters for contact name, phone and email 
+  - The function that handles the form submission
+- Render a `form` with:
+  - The `onSubmit` attribute set
+  - 3 controlled `input` components, one for each piece of contact data
+  - A submit button
+- Include a `pattern` attribute to the phone `input` with [a regex](https://regexlib.com/Search.aspx?k=phone&c=-1&m=-1&ps=20) that matches the phone locale of your preference.
+
+### Hint
+
+- Make `props` a function parameter
+- Use `<form onSubmit={props.handlerFunction} >` to create the html form
+- The children of the form should be `<input>`s of `type` `text`, `tel`, `email` and `submit`
+- The `value` attribute of the 3 controlled `input`s should be set to the associated variable in `props`
+- The `onChange` attributes of the 3 controlled `input`s should be set to a callback function that passes `e.target.value` to the the associated setter function
+- One example of a phone regex is the US locale: `"[1-9][0-9]{2}-[1-9][0-9]{2}-[0-9]{4}"`
+
+### Task #5 
+
+Based on the given requirements, implement `TileList` as a stateless component that renders a list of `Tile` components using an array of objects.
+
+This component will:
+
+- Have one prop:
+  - An array to render as a list
+- Use the array in props to iteratively render `Tile` components.
+- Add the objects in the array to the props of the `Tile` component.
+
+### Hint
+
+- Import the `Tile` component
+- Make `props` a function parameter
+- Use the `map()` method on the array prop
+- The `map()` callback function has a data object and an index as parameters
+- The callback should return a `Tile` component with the data object added to `props` and set the component `key` to the index 
+
+### Task #6 
+
+Based on the given requirements, implement `Tile` as a stateless component that renders the data from an object.
```

Comment:
If you add the explanation of Tiles in the previous task, this may not be necessary, but it might be nice to re-iterate that the Tile is a generic component used to render either Contact information or appointment information